src/Microsoft.ML.Data/Prediction/CalibratorCatalog.cs

```diff
@@ -122,6 +122,12 @@ SchemaShape IEstimator<CalibratorTransformer<TICalibrator>>.GetOutputSchema(Sche
             return new SchemaShape(outColumns.Values);
         }
 
+        /// <summary>
+        /// Fits the scored <see cref="IDataView"/> creating a <see cref="CalibratorTransformer{TICalibrator}"/>.
+        /// </summary>
+        /// <param name="input"></param>
+        /// <returns>A trained <see cref="CalibratorTransformer{TICalibrator}"/> that will transforms the data by adding the
+        /// Probability column.</returns>
```

Comment:
Do we have a constant string for the name of a probability column? If yes, it'd be nice to make a reference here (<see cref="">.